integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -152,18 +152,19 @@ describe.skipIf(environment.missingServer, "Flexible sync", function () {
         }).to.throw("'partitionValue' cannot be specified when flexible sync is enabled");
       });
 
-      it("accepts { flexible: false } and a partition value", function () {
-        expect(() => {
-          new Realm({
-            sync: {
-              _sessionStopPolicy: SessionStopPolicy.Immediately,
-              flexible: false,
-              user: this.user,
-              partitionValue: "test",
-            },
-          });
-        }).to.not.throw();
-      });
+      // NOTE: This is not a compatible configuration anymore and will cause a typescript error
```

Comment:
`NIT`: Can it be removed?

Reply:
I'll do what Tom suggests here.

Reply:
Using [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) is even better for this use-case as it will fail if no error was raised.

Reply:
Good idea.  That is now implemented.